src/requirements/data/colleges/asFA2020.ts

```diff
@@ -1,24 +1,42 @@
 import { Course, CollegeOrMajorRequirement } from '../../types';
-import { courseIsFWS, ifCodeMatch, courseIsForeignLang } from '../checkers-common';
+import {
+  courseIsFWS,
+  ifCodeMatch,
+  courseIsForeignLang,
+  courseCountsForASCredit,
+} from '../checkers-common';
 
 const casFA2020Requirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: '120 Total Credits',
+    description:
+      'Students must earn a minimum of 120 academic credits, which may include AP/test credits. ' +
+      'However, courses in military training, service as a teaching assistant, physical education, remedial or developmental training, ' +
+      'precalculus mathematics, supplemental science and mathematics, and offered by the Learning Strategies Center and English as a second language do not count. ' +
+      'The College\'s "Courses That Won\'t Count" website has a full list of excluded courses.',
+    source: 'https://as.cornell.edu/education/degree-requirements',
+    checker: [(course: Course): boolean => courseCountsForASCredit(course)],
+    fulfilledBy: 'credits',
+    perSlotMinCount: [120],
+    allowCourseDoubleCounting: true,
```

Comment:
There is code somewhere that makes the pre-FA2020 check to not show all of the courses that fulfill it - because the list would be very long and it is essentially just a long list of everything in your semester plan. Could you do the same here?

![image](https://user-images.githubusercontent.com/25535093/165101170-8043e37c-59c8-485c-8b26-d21064e00f62.png)

Reply:
yup, just figured that out